Runtime/UIElements/Controls/UIUtils.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+static class UIUtils
+    {
+        static readonly string[] s_SizeUnits = { "KB", "MB", "GB", "TB" };
+
+        public static void SetElementDisplay(VisualElement element, bool value)
+        {
+            if (element == null)
+                return;
+
+            element.style.display = value ? DisplayStyle.Flex : DisplayStyle.None;
+            element.style.visibility = value ? Visibility.Visible : Visibility.Hidden;
+        }
+
+        public static void SetElementDisplayNonEmpty(TextElement element)
+        {
+            if (element == null)
+                return;
+
+            var nonEmpty = !String.IsNullOrEmpty(element.text);
+            element.style.display = nonEmpty ? DisplayStyle.Flex : DisplayStyle.None;
+            element.style.visibility = nonEmpty ? Visibility.Visible : Visibility.Hidden;
+        }
+
+        public static bool IsElementVisible(VisualElement element)
```

Comment:
Same comment about method semantics. The fact that we have `Visibility.Visible` and `DisplayStyle.Flex` does not mean that element is actually "Visible"